pom.xml

```diff
@@ -92,32 +90,30 @@
 			<optional>true</optional>
 		</dependency>
 		<dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>claim</artifactId>
-            <version>2.3</version>
-            <optional>true</optional>
-        </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>config-file-provider</artifactId>
-            <version>2.7.1</version>
-            <optional>true</optional>
-        </dependency>
-        <dependency>
-        	<groupId>org.codehaus.plexus</groupId>
-        	<artifactId>plexus-utils</artifactId>
-        	<version>2.0.5</version>
-        	<scope>test</scope>
-        </dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>claim</artifactId>
+			<version>2.3</version>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>config-file-provider</artifactId>
+			<version>2.7.1</version>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>org.codehaus.plexus</groupId>
+			<artifactId>plexus-utils</artifactId>
+			<version>2.0.5</version>
+			<scope>test</scope>
+		</dependency>
 	</dependencies>
-
-    <scm>
-        <connection>scm:git:git@github.com:jenkinsci/configurationslicing-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/configurationslicing-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/configurationslicing-plugin</url>
-      <tag>HEAD</tag>
-  </scm>
-
+	<scm>
+		<connection>scm:git:git@github.com:jenkinsci/configurationslicing-plugin.git</connection>
+		<developerConnection>scm:git:git@github.com:jenkinsci/configurationslicing-plugin.git</developerConnection>
+		<url>https://github.com/jenkinsci/configurationslicing-plugin</url>
+		<tag>HEAD</tag>
+	</scm>
```

Comment:
Hi!
Those changes above IMO shouldn't be there. All seem to be whitespace/formatting changes, unrelated to the goal of that PR.